app/controllers/workflows/emails_controller.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+class Workflows::EmailsController < Sellers::BaseController
+  before_action :set_workflow
+  before_action :authorize_workflow
+
+  layout "inertia"
+
+  FLASH_CHANGES_SAVED = "Changes saved!"
+  FLASH_WORKFLOW_PUBLISHED = "Workflow published!"
+  FLASH_WORKFLOW_UNPUBLISHED = "Unpublished!"
+
+  inertia_share do
+    RenderingExtension.custom_context(view_context).merge(
+      current_user: current_user_props(current_user, impersonated_user),
+      authenticity_token: form_authenticity_token,
+      flash: inertia_flash_props,
+      title: @title
+    )
+  end
```

Comment:
You already have all of this available via the module `InertiaRendering` that is included in `ApplicationController`